src/CashTransaction.php

```diff
@@ -0,0 +1,127 @@
+<?php
```

Comment:
is this (partially) copied from another class? Please see if you can move some code into traits then.

Reply:
Yes it is. I will look into it!

Reply:
@willemstuursma The reason why I copied some code is because the methods *getCurrency* and *setCurrency* of the trait *StartAndCloseValueFields* are not compatible with the methods of *PhpTwinfield\BaseTransaction*. If I change the methods in *BaseTransaction* I'm afraid I will break existing implementations. Any idea how to solve this?

Reply:
Oh that sucks! It doesn't use any Currency value objects. 

To me, updating `PhpTwinfield\BaseTransaction` is fine. We'll release a new minor version with a changelog instead of a bugfix version.